### PR TITLE
Vet Clinic Databae: Updating animals

### PR DESCRIPTION
**Pull request by misalazeem**
- Added transactions
- Updated tables
- Added new queries

Query 1: 
image

Query 2: 
image

Query 3:
image

Query 4:
image

Query 5:
image

Query 6:
image

Query 7:
image

Query 8:
image

Query 9:
image

Query 10:
image

Post Review Required screenshots:

1.
image

2.
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -28,4 +28,64 @@ WHERE name != 'Gabumon';
...
-WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-----
+BEGIN;
+    UPDATE animals
+SET species = 'unspecified';
+    ROLLBACK;
+END;
```
Comment (omar-muhamad) [inline queries.sql]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=to%20unspecified.-,Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.,-Inside%20a%20transaction), Please verify that change was made and verify that the species columns went back after the rollback.

Hint: 
```sql
   BEGIN; 
   UPDATE animals
   SET species = 'unspecified'; 
   SELECT species from animals; -- verify that change was made
   ROLLBACK;
   SELECT species from animals; -- verify that change was undone
```

File: queries.sql [hunk 2]
```diff
@@ -28,4 +28,64 @@ WHERE name != 'Gabumon';
...
+
+-----
+BEGIN;
+    UPDATE animals SET species = 'digimon' WHERE name like '%mon';
+    UPDATE animals SET species = 'pokemon' WHERE species is null;
+    COMMIT;
+END;
```
Comment (omar-muhamad) [inline queries.sql:49]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=species%20already%20set.-,Verify%20that%20changes%20were%20made.,Verify%20that%20changes%20persist%20after%20commit.,-Take%20a%20screenshot), Please verify that changes were made and verify that changes persist after commit.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @misalazeem,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (muneebulrehman) [reply queries.sql]: As requested by the previous reviewer, kindly verify whether changes were made and verify specifies column was set back to before setting it to `unspecified`. It is a part of the project requirement.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @misalazeem , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Descriptive PR
- Data inserted successfully

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by misalazeem** [comment]
> Hi @misalazeem ,
> 
> Great work on making the changes requested by a previous reviewer 👏 You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
> 
> ## Highlights
> * Descriptive PR
> * Data inserted successfully
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

If you look at the description i added new screenshots

image

Kindly verify it.

**Review by Lameck1 — COMMENTED** [review]
Hello @misalazeem👋,

_This is an additional review as a result of the second opinion request._

There is a misunderstanding I would like to clarify. The CRs are not talking about you just adding a screenshot, they would like to see a change in the code and this is in line with the project requirements.

@Omar-Muhamad had you given the suggestion quoted below but it seems you not considered it. This is where the bone of contention is:

> * [ ]  As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=to%20unspecified.-,Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.,-Inside%20a%20transaction), Please verify that change was made and verify that the species columns went back after the rollback.
> 
> Hint:
> 
> ```sql
>    BEGIN; 
>    UPDATE animals
>    SET species = 'unspecified'; 
>    SELECT species from animals; -- verify that change was made
>    ROLLBACK;
>    SELECT species from animals; -- verify that change was undone
> ```

Take a look at the comments above, that state, "-- verify that change was undone". It is that simple line of code that you need to add to do away with the hold-up. That is:

```sql
SELECT species from animals;
```

Before Rollaback and after Rollaback. Just that.

Please work on the above suggestion and request for another code review.

Keep rocking 🚀!

**Review by mukhammadrizooff — APPROVED** [review]
Hi @misalazeem, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**